### PR TITLE
Create a sample and docs for RMAN backup schedules

### PR DESCRIPTION
**Pull request by mfielding**
Extending the existing samples and documentation to cover RMAN backup scheduling.

### CODE REVIEW

File: docs/content/backup-restore/rman-backups.md [hunk 1]
```diff
@@ -100,6 +100,83 @@ rman3-inst-opts   mydb            Physical      Instance         4
 
 Once the backup phase changed to `Succeeded`, the physical backup creation is complete and ready to use.
 
+## Scheduling regular backups
+
+El Carro can manage schedules and retention periods for RMAN backups.
```
Comment (akinfermo) [inline docs/content/backup-restore/rman-backups.md:105]:
```suggestion
El Carro can manage schedules and retention periods for backups (RMAN and Snapshot).
```

File: docs/content/backup-restore/rman-backups.md [hunk 2]
```diff
@@ -100,6 +100,83 @@ rman3-inst-opts   mydb            Physical      Instance         4
...
+
+### Prepare the backup schedule manifest
+
+To schedule a backup, create a manifest of type `BackupSchedule` with the following fields:
+
+* backupSpec: the parameters for the backup itself, as for a [one-off backup](#preparing-a-backup-cr-manifest)
```
Comment (akinfermo) [inline docs/content/backup-restore/rman-backups.md:111]: I like that you provided a thorough descriptions of these fields. We should do more of that in our docs.

**Review by akinfermo — CHANGES_REQUESTED** [review]

File: docs/content/backup-restore/rman-backups.md (hunk 2 above)
Comment (mfielding) [reply docs/content/backup-restore/rman-backups.md:111]: In this case I'm trying to follow the existing structure and style of the doc.  The canonical location for this type of documentation is actually the CRD: https://github.com/GoogleCloudPlatform/elcarro-oracle-operator/blob/77fdd40aa416d8b76f813aa0e598ee731ab05286/oracle/config/crd/bases/oracle.db.anthosapis.com_backupschedules.yaml.

**Comment by mfielding** [comment]
Why the force push?  GitHub's "apply suggested changes" functionality added a `Co-authored-by` line using the gmail address from akinferno@'s public profile.  This address failed the CLA check.  Pushing an amended commit that has their google.com address.

**Review by akinfermo — APPROVED** [review]
LGTM. Thanks for improving our documentation!

**Pull request merged**